src/components/Home/HomePresentation.js

```diff
@@ -94,19 +104,19 @@ function HomePresentation({ backgroundId, backgroundType, site }) {
             className="bg-gradient-to-b from-word to-word-dark"
           >
             <div className="flex flex-col md:flex-row items-center justify-center mx-auto max-w-7xl p-3 md:p-5">
-              <div className="w-48 flex flex-col items-center justify-center">
+              <div
+                className={`${logoWidth} flex flex-col items-center justify-center`}
+              >
                 <div className="flex mb-2">
                   <LazyImage
-                    imgStyling="h-auto w-24 sm:w-32 lg:w-40 rounded-full"
+                    imgStyling={`${logoHeight} w-full rounded-full`}
```

Comment:
Should this one be logoWidth? Does it matter?

Reply:
The width for the logo is being set in the parent div on line 108 so that the Title can match it - w-full on line 112 just ensures that the logo fills the width of the parent div.